<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerConfigWidget</class>
 <widget class="QWidget" name="PlayerConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>644</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="m_grpGeneral">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,0">
      <item row="1" column="1">
       <widget class="QCheckBox" name="kcfg_UnpauseOnDoubleClick">
        <property name="text">
         <string>Resume playback after line double click</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="lab_SeekJumpLength">
        <property name="text">
         <string>Seconds to &amp;jump on seek:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_SeekJumpLength</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="kcfg_StepJumpLength">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="kcfg_VideoAutoPlay">
        <property name="text">
         <string>Start playing video when loaded</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="kcfg_ShowPositionTimeEdit">
        <property name="text">
         <string>Show editable position control</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="lab_StepJumpLength">
        <property name="text">
         <string>Fra&amp;me step amount:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_StepJumpLength</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="lab_VolumeAmplification">
        <property name="text">
         <string>Volume amplification:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSlider" name="kcfg_VolumeAmplification">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>30</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="pageStep">
         <number>3</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="kcfg_SeekJumpLength">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>300.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_grpSubtitles">
     <property name="title">
      <string>Subtitles</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,0">
      <item row="4" column="1">
       <widget class="QSpinBox" name="kcfg_OutlineWidth">
        <property name="suffix">
         <string notr="true"/>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="lab_OutlineColor">
        <property name="text">
         <string>Outline color:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_OutlineColor</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="KColorCombo" name="kcfg_OutlineColor"/>
      </item>
      <item row="2" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="lab_FontColor">
        <property name="text">
         <string>Te&amp;xt color:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_FontColor</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KColorCombo" name="kcfg_FontColor"/>
      </item>
      <item row="0" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="lab_FontFamily">
        <property name="text">
         <string>Font family:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_FontFamily</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFontComboBox" name="kcfg_FontFamily"/>
      </item>
      <item row="4" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="lab_OutlineWidth">
        <property name="text">
         <string>Outline width:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_OutlineWidth</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="lab_FontSize">
        <property name="text">
         <string>Font si&amp;ze:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_FontSize</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="kcfg_FontSize"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_grpPreview">
     <property name="title">
      <string>Subtitles Preview</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="SubtitleComposer::TextOverlayWidget" name="m_textOverlayWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>2</horstretch>
          <verstretch>2</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorCombo</class>
   <extends>QComboBox</extends>
   <header>kcolorcombo.h</header>
  </customwidget>
  <customwidget>
   <class>SubtitleComposer::TextOverlayWidget</class>
   <extends>QWidget</extends>
   <header>textoverlaywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>kcfg_VideoAutoPlay</tabstop>
  <tabstop>kcfg_UnpauseOnDoubleClick</tabstop>
  <tabstop>kcfg_SeekJumpLength</tabstop>
  <tabstop>kcfg_StepJumpLength</tabstop>
  <tabstop>kcfg_ShowPositionTimeEdit</tabstop>
  <tabstop>kcfg_FontFamily</tabstop>
  <tabstop>kcfg_FontSize</tabstop>
  <tabstop>kcfg_FontColor</tabstop>
  <tabstop>kcfg_OutlineColor</tabstop>
  <tabstop>kcfg_OutlineWidth</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
